Wenab/Common/LittleByte.Common/AspNet/Configuration/HealthCheckConfiguration.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+using LittleByte.Common.AspNet.Json;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace LittleByte.Common.AspNet.Configuration
+{
+    public static class HealthCheckConfiguration
```

Comment:
It's common for services to provide a health check endpoint. It can be useful for:

1. Users of a service to check if the service is up
2. Testing new deployments to make sure they are up and running and potentially backing out a deployment if the health check fails